app/Http/Controllers/EpisodeController.php

```diff
@@ -41,8 +41,9 @@ public function index()
     public function allEpisode()
     {
         $episodes = Episode::get();
+	    $channels = Channel::all();
```

Comment:
**Code  Quality - Nitpick*

Please convert your tabs to spaces in your IDE & use 4 spaces for 1 tab